blueprint/api.apib

```diff
@@ -2624,6 +2597,33 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + id: `1` (string, required)
 + type: `stripe-customer`
 
+## Stripe Plan Attributes (object)
++ amount: 999 (number) - Describes the amount of a plan in cents.
++ created: `2016-07-08T03:03:51.967Z` (string)
```

Comment:
From 383:
`+  * `created` - A timestamp, indicating when the plan was created by Stripe
`
I'll add the comment into this also.